src/web/lib/components/ThemeColorsEditor/index.scss

```diff
@@ -17,13 +17,15 @@
       width: 36px;
 
       &.disabled {
-        opacity: 0.4;
+        opacity: .4;
       }
+
       &:hover {
-        background: rgba(12, 12, 13, 0.1);
+        background: rgba(12, 12, 13, .1);
       }
+
       &:active {
-        background: rgba(12, 12, 13, 0.2);
+        background: rgba(12, 12, 13, .2);
```

Comment:
We seem to use this `rgba(12, 12, 13, n)` color quite a bit (6-7 places at least). Not sure if it's worth extracting out into some Sass variable so we could do `rgba($light-blue, .2)` or something. Possibly/probably not worth the effort though.